src/ol_infrastructure/applications/forum/__main__.py

```diff
@@ -0,0 +1,235 @@
+"""Create the resources needed to run a forum server.
+"""
+import base64
+import json
+import textwrap
+from pathlib import Path
+
+import pulumi_vault as vault
+import yaml
+from pulumi import Config, StackReference
+from pulumi_aws import ec2, get_caller_identity, iam
+
+from bridge.lib.magic_numbers import FORUM_SERVICE_PORT
+from bridge.secrets.sops import read_yaml_secrets
+from ol_infrastructure.components.aws.auto_scale_group import (
+    BlockDeviceMapping,
+    OLAutoScaleGroupConfig,
+    OLAutoScaling,
+    OLLaunchTemplateConfig,
+    TagSpecification,
+)
+from ol_infrastructure.lib.aws.ec2_helper import InstanceTypes, default_egress_args
+from ol_infrastructure.lib.consul import get_consul_provider
+from ol_infrastructure.lib.ol_types import AWSBase
+from ol_infrastructure.lib.pulumi_helper import parse_stack
+from ol_infrastructure.lib.vault import setup_vault_provider
+
+stack_info = parse_stack()
+forum_config = Config("forum")
+if Config("vault").get("address"):
+    setup_vault_provider()
+network_stack = StackReference(f"infrastructure.aws.network.{stack_info.name}")
+policy_stack = StackReference("infrastructure.aws.policies")
+dns_stack = StackReference("infrastructure.aws.dns")
+consul_stack = StackReference(
+    f"infrastructure.consul.{stack_info.env_prefix}.{stack_info.name}"
+)
+kms_stack = StackReference(f"infrastructure.aws.kms.{stack_info.name}")
```

Comment:
I don't think this is getting used anywhere so it should get deleted to avoid confusion.